### PR TITLE
Add new Testnets

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/goerli/info/info.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Goerli",
```
Comment (vcoolish) [inline blockchains/goerli/info/info.json:2]: add Testnet to each name as on example

File: blockchains/okxt/info/info.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "OKB Testnet",
```
Comment (vcoolish) [inline blockchains/okxt/info/info.json:2]: please add this chain main net in separate pr

File: blockchains/tftm/info/info.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Fantom Testnet",
+    "symbol": "tFTM",
+    "type": "coin",
+    "decimals": 18,
+    "coin_type": 10000250,
+    "description": "Fantom is a high-performance, scalable, customizable, and secure smart-contract platform. It is designed to overcome the limitations of previous generation blockchain platforms. Fantom is permissionless, decentralized, and open-source.",
+    "website": "http://fantom.foundation",
+    "explorer": "https://testnet.ftmscan.com",
```
Comment (vcoolish) [inline blockchains/tftm/info/info.json:9]: rpc missing

File: blockchains/sepolia/info/info.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Sepolia Testnet",
+    "website": "https://sepolia.dev",
+    "description": "Sepolia was designed to simulate harsh network conditions, and has shorter block times, which enable faster transaction confirmation times and feedback for developers.",
+    "explorer": "https://sepolia.etherscan.io",
+     "symbol": "ETH",
```
Comment (vcoolish) [inline blockchains/sepolia/info/info.json:6]: formatting

File: blockchains/polygonmumbai/info/info.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Polygon Mumbai Testnet",
+    "website": "https://polygon.technology",
+    "description": "The Alchemy Goerli faucet is free, fast, and does not require authentication, though you can optionally login to Alchemy to get an increased drip.",
+    "explorer": "https://mumbai.polygonscan.com/",
+     "symbol": "MATIC",
```
Comment (vcoolish) [inline blockchains/polygonmumbai/info/info.json:6]: formatting

Comment (vcoolish) [reply blockchains/polygonmumbai/info/info.json:6]: check on all files

File: blockchains/ethereum/chainlist.json [hunk 6]
```diff
@@ -87,6 +87,46 @@
...
+         "chain": "10000070",
+         "coinId": "optimismgoerli"
+      },
+      {
+         "chain": "11155111",
+         "coinId": "sepolia"
+      },
+      {
+         "chain": "66",
+         "coinId": "okbtestnet"
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:129]: what is this coin id?

File: blockchains/ethereum/chainlist.json [hunk 7]
```diff
@@ -87,6 +87,46 @@
...
+      {
+         "chain": "1287",
+         "coinId": "moonbasealpha"
+      },
+      {
+         "chain": "10000250",
+         "coinId": "fantomtestnet"
+      },
+      {
+         "chain": "10000070",
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:120]: 420

File: blockchains/ethereum/chainlist.json [hunk 8]
```diff
@@ -87,6 +87,46 @@
...
+      {
+         "chain": "324",
+         "coinId": "zksyncera"
+      },
+      {
+         "chain": "60",
+         "coinId": "goerli"
+      },
+      {
+         "chain": "10042221",
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:104]: 421613

File: blockchains/arbitrumgoerli/info/info.json [hunk 9]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Arbitrum Goerli Testnet",
+    "website": "https://goerlifaucet.com",
+    "description": "The Alchemy Goerli faucet is free, fast, and does not require authentication, though you can optionally login to Alchemy to get an increased drip.",
+    "explorer": "https://goerli.arbiscan.io/",
+    "research": "https://goerli.net/#about",
+     "symbol": "ETH",
```
Comment (vcoolish) [inline blockchains/arbitrumgoerli/info/info.json:7]: AGOR

File: blockchains/ethereum/chainlist.json [hunk 10]
```diff
@@ -87,6 +87,46 @@
...
+         "coinId": "arbitrumgoerli"
+      },
+      {
+         "chain": "966",
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:108]: 80001

File: blockchains/ethereum/chainlist.json [hunk 11]
```diff
@@ -87,6 +87,46 @@
...
+         "coinId": "polygonmumbai"
+      },
+      {
+         "chain": "1287",
+         "coinId": "moonbasealpha"
+      },
+      {
+         "chain": "10000250",
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:116]: 4002

File: blockchains/ethereum/chainlist.json [hunk 12]
```diff
@@ -87,6 +87,46 @@
...
+      {
+         "chain": "1422",
+         "coinId": "polygonzkevm"
+      },
+      {
+         "chain": "324",
+         "coinId": "zksyncera"
+      },
+      {
+         "chain": "60",
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:100]: 5

File: blockchains/ethereum/chainlist.json [hunk 13]
```diff
@@ -87,6 +87,46 @@
       {
          "chain": "2222",
          "coinId": "kavaevm"
+      },
+      {
+         "chain": "1422",
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:92]: 1101

**Comment by cryptomanz1** [comment]
#24604

File: blockchains/avalanchecfuji/info/info.json [hunk 14]
```diff
@@ -0,0 +1,12 @@
...
+    "website": "https://avax.network",
+    "description": "Avalanche is an open-source platform for launching Decentralized Finance (DeFi) applications and enterprise blockchain deployments in an interoperable, highly scalable ecosystem.",
+    "explorer": "https://testnet.snowtrace.io/",
+     "symbol": "AVAX",
+     "rpc_url": "https://api.avax-test.network/ext/bc/C/rpc",
+     "type": "coin",
+     "coin_type": 43113,
+     "decimals": 18,
+     "status": "active"
+ }
```
Comment (saviourini9) [inline blockchains/avalanchecfuji/info/info.json]: blockchains/avalanchecfuji/info/logo.png

File: blockchains/tftm/info/info.json (hunk 3 above)
Comment (saviourini9) [reply blockchains/tftm/info/info.json:9]: [export-token-0x38ec854248a0f6a81aad2081c11fc56b74471c56.2.csv](https://github.com/trustwallet/assets/files/11184938/export-token-0x38ec854248a0f6a81aad2081c11fc56b74471c56.2.csv)

File: blockchains/bnbt/info/info.json [hunk 15]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Binance Smart Chain Testnet",
+    "website": "https://testnet.binance.org/",
+    "description": "Fast and secure decentralized digital asset exchange",
+    "explorer": "https://testnet.bscscan.com",
+    "symbol": "tBNB",
+    "rpc_url": "https://data-seed-prebsc-1-s1.binance.org:8545",
+    "type": "coin",
+    "coin_type": 20000714,
```
Comment (vcoolish) [inline blockchains/bnbt/info/info.json:9]: this is id from main net

File: blockchains/goerlitestnet/info/info.json [hunk 16]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Goerli Testnet",
+    "website": "https://goerlifaucet.com",
+    "description": "The Alchemy Goerli faucet is free, fast, and does not require authentication, though you can optionally login to Alchemy to get an increased drip.",
+    "explorer": "https://goerli.etherscan.io",
+    "research": "https://goerli.net/#about",
+    "symbol": "ETH",
+    "rpc_url": "https://rpc.goerli.mudit.blog",
+    "type": "coin",
+    "coin_type": 60,
```
Comment (vcoolish) [inline blockchains/goerlitestnet/info/info.json:10]: this coin type from main net

File: blockchains/ethereum/chainlist.json [hunk 17]
```diff
@@ -87,6 +87,42 @@
...
+         "chain": "80001",
+         "coinId": "polygonmumbai"
+      },
+      {
+         "chain": "1287",
+         "coinId": "moonbasealpha"
+      },
+      {
+         "chain": "4002",
+         "coinId": "fantomtestnet"
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:117]: coin id should match folder name, please check one by one

**Comment by Shaylavae** [comment]
>  @#`
Duplicate of #`

**Review by Shaylavae — COMMENTED** [review]
#24614

**Comment by saviourini9** [comment]
[Blazing-The-WayTrue-Stories-Songs-and-Sketches-of-Puget-Sound_39334-0.1.0.tar.gz](https://github.com/trustwallet/assets/files/11188994/Blazing-The-WayTrue-Stories-Songs-and-Sketches-of-Puget-Sound_39334-0.1.0.tar.gz)
[Uploading go-ethereum-1.11.5 2.tar.gz…]()

**Comment by saviourini9** [comment]
[Blazing-The-WayTrue-Stories-Songs-and-Sketches-of-Puget-Sound_39334-0.1.0.tar.gz](https://github.com/trustwallet/assets/files/11188994/Blazing-The-WayTrue-Stories-Songs-and-Sketches-of-Puget-Sound_39334-0.1.0.tar.gz)
[go-ethereum-1.11.5 2.tar.gz](https://github.com/trustwallet/assets/files/11188996/go-ethereum-1.11.5.2.tar.gz)

**Comment by saviourini9** [comment]
[export-token-0x38ec854248a0f6a81aad2081c11fc56b74471c56.csv](https://github.com/trustwallet/assets/files/11189001/export-token-0x38ec854248a0f6a81aad2081c11fc56b74471c56.csv)

**Comment by saviourini9** [comment]
[Uploading AccountStatement547038 Medium.zip…]()

**Comment by saviourini9** [comment]
[AccountStatement547038 Medium.zip](https://github.com/trustwallet/assets/files/11189008/AccountStatement547038.Medium.zip)

**Comment by saviourini9** [comment]
[export-token-0x38ec854248a0f6a81aad2081c11fc56b74471c56.zip](https://github.com/trustwallet/assets/files/11189014/export-token-0x38ec854248a0f6a81aad2081c11fc56b74471c56.zip)
[export-token-0x38ec854248a0f6a81aad2081c11fc56b74471c56 2.csv](https://github.com/trustwallet/assets/files/11189015/export-token-0x38ec854248a0f6a81aad2081c11fc56b74471c56.2.csv)

File: blockchains/polygonmumbai/info/info.json [hunk 18]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Polygon Mumbai Testnet",
+    "website": "https://polygon.technology",
+    "description": "The Alchemy Goerli faucet is free, fast, and does not require authentication, though you can optionally login to Alchemy to get an increased drip.",
+    "explorer": "https://mumbai.polygonscan.com/",
+    "symbol": "MATIC",
+    "rpc_url": "https://goerli-rollup.arbitrum.io/rpc",
```
Comment (vcoolish) [inline blockchains/polygonmumbai/info/info.json:7]: wrong rpc 
https://matic-mumbai.chainstacklabs.com

File: blockchains/zksync/info/info.json [hunk 19]
```diff
@@ -6,7 +6,6 @@
     "research": "https://github.com/zksync-sdk",
     "symbol": "ETH",
     "rpc_url": "https://zksync2-mainnet.zksync.io",
-    "coin_type": 324,
```
Comment (vcoolish) [inline blockchains/zksync/info/info.json:9]: return this, it's not testate

**Review by vcoolish — APPROVED** [review]

**Pull request merged**